modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/transfer/ProcessImport.java

```diff
@@ -78,72 +78,76 @@ public void importProcesses(InputStream processZipInputStream, String user)
             byte[] buffer = new byte[2048];
             ZipInputStream zipInputStream = new ZipInputStream(processZipInputStream);
             File importsDir = new File(ProcessCenterConstants.IMPORTS_DIR);
-            if (importsDir != null) {
-                importsDir.mkdirs();
-
-                try {
-                    ZipEntry entry;
-                    while ((entry = zipInputStream.getNextEntry()) != null) {
-                        //counter++;
-                        String outpath = ProcessCenterConstants.IMPORTS_DIR + "/" + entry.getName();
-                        String dirPath = outpath.substring(0, outpath.lastIndexOf("/"));
-                        new File(dirPath).mkdirs();
-                        FileOutputStream fileOutputStream = null;
-                        try {
-                            fileOutputStream = new FileOutputStream(outpath);
-                            int len = 0;
-                            while ((len = zipInputStream.read(buffer)) > 0) {
-                                fileOutputStream.write(buffer, 0, len);
-                            }
-                        } finally {
-                            if (fileOutputStream != null) {
-                                fileOutputStream.close();
+            try {
+                if (importsDir != null) {
+                    importsDir.mkdirs();
+
+                    try {
+                        ZipEntry entry;
+                        while ((entry = zipInputStream.getNextEntry()) != null) {
+                            //counter++;
+                            String outpath = ProcessCenterConstants.IMPORTS_DIR + "/" + entry.getName();
```

Comment:
Fixed in https://github.com/wso2/product-pc/commit/f93f5b6b219c403c6f5fe69b4cbb16f2e419fca5